File: Minimize Maximum Pair Sum in Array.js
var minPairSum = function(nums) {
     let sum = 0;
   nums.sort((a,b) => a - b);
   
   let half = nums.length/2;
    
   
 for(let i = 0,j=nums.length-1 ;i<half&&j>=half;i++,j--){
  sum=Math.max(sum,nums[i]+nums[j])
 }
    
    return sum;
};

File: 1988-minimize-maximum-pair-sum-in-array/minimize-maximum-pair-sum-in-array.py
class Solution(object):
    def minPairSum(self, nums):
        nums.sort()
        i = 0
        j = len(nums) - 1
        max_sum = 0
        while i <=j:
            max_sum = max(nums[i]+nums[j] ,max_sum)
            
            i+=1
            j-=1
        return max_sum
      

        